test/unit/job_test.rb

```diff
@@ -1288,7 +1290,7 @@
           assert_equal 2, Sidekiq::Cron::Job.all('*').size, 'Should have 2 jobs'
 
           Sidekiq::Cron.configuration.available_namespaces = [custom_namespace]
-          assert_equal 1, Sidekiq::Cron::Job.all('*').size, 'Should have 1 job'
+          assert_equal 3, Sidekiq::Cron::Job.all('*').size, 'Should have 3 job'
```

Comment:
@viralpraxis Could you please explain why this assert changes here?

Reply:
In https://github.com/sidekiq-cron/sidekiq-cron/pull/551 we made sure that the default namespace is always included in the namespaces list. Thus, `Sidekiq::Cron::Job.all('*')` **should** also include default jobs -- but that patch still relied on `Sidekiq::Cron.configuration.available_namespaces` instead of `Sidekiq::Cron::Namespace.all`. I can open another PR (as a follow-up to https://github.com/sidekiq-cron/sidekiq-cron/pull/551) if there's any confusion

Reply:
In other words, [this diff](https://github.com/sidekiq-cron/sidekiq-cron/pull/552/files#diff-45ac47a9faa0840538f2cae00fdd31667a33fb6b4f94a7428d9c80bceaf8a87eL696-R696) should have been included in #551

Reply:
Thanks for explaining!